modules/linux/main.tf

```diff
@@ -86,7 +87,7 @@ locals {
     var.app_settings
   )
   tags = merge(
-    { service = local.datadog_service },
+    { service = local.datadog_service, dd_sls_terraform_module = local.module_version },
```

Comment:
Do we need a way to differentiate the windows module from the linux module? It might be nice to have that usage breakdown